couchbase-ansible/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Advanced Linux : Deploy a Multi VM Couchbase Cluster with Ansible",
+  "description": "This advanced template deploys N  Linux VMs (CentsOS or Ubuntu) and it configures a Multi VM Couchbase Cluster.",
+  "summary": "This advanced template deploys a Multi VM Couchbase cluster.It uses the Ansible Playbooks provided by couchbaselabs in the GitHub repo ansible-couchbase-server to deploy Couchbase.",
```

Comment:
Summary looks too long according to the validation character limit, but description looks right if you just want to switch them